### ISSUE TITLE
Using arrow keys to navigate tree view

### ISSUE THREAD

**Issue by digama0** [Baezon/pof-tools#24]
I would have expected that if I click on a tree item then I can use the arrow keys to go up and down to focus (not select) other elements in the tree view, but it does not seem to work. You can use tab and shift tab to select list items, but clicking resets the focus to the beginning again.

**Comment by Baezon** [comment]
Yes, this is planned, just requires a bit of manual work. Doesn't come for free with egui like several other keyboard shortcuts.
